react/the_react_ecosystem/styling_react_applications.md

```diff
@@ -0,0 +1,66 @@
+### Introduction
+
+Up until now you've been using vanilla CSS to style your React projects, but there are other options available. This lesson is intended to be more like a guide or a list of options for you to explore.  
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- Know about ways to style React applications 
+
+### Vanilla CSS 
+
+Vanilla CSS is the simplest way to style. In the previous courses, you've learned a lot of CSS and all of those skills are applicable to React. There are a couple of things we'd like to highlight.
+
+1. CSS Modules
+
+    CSS Modules let you write CSS style declarations that are scoped locally.
+
+1. CSS Utility Frameworks
+
+    CSS frameworks are popular choice of styling React applications. They provide a set of pre-defined classes that you can directly use in your HTML, or JSX in our case. [Tailwind CSS](https://tailwindcss.com) is by far the most popular choice. 
```

Comment:
```suggestion
Vanilla CSS is the simplest way to style. In the previous courses, you've learned a lot of CSS and all of those skills are applicable to React. There are a couple of things we'd like to highlight.

1. `CSS Modules` let you write CSS style declarations that are scoped locally.
1. `CSS Utility Frameworks` are a popular choice for styling React applications. They provide a set of pre-defined classes that you can directly use in your HTML, or JSX in our case. [Tailwind CSS](https://tailwindcss.com) is by far the most popular choice. 
```

The styling/layout of this seems "off" to me even though I know it adheres to the syleguide. I think the explanation just don't need their own subsection. I proposed something else but you may know of something better or want to leave it as is. 

do mind the typo that a fixed in the second list item.

Reply:
lgtm